src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/LinkServiceImpl.java

```diff
@@ -324,32 +346,87 @@ private BatchResults getLinkPropsResult(List<LinkPropsResponse> responses, List<
         return messagingChannel.sendAndGetChunked(nbworkerTopic, message)
                 .thenApply(response -> response.stream()
                         .map(IslInfoData.class::cast)
-                        .map(linkMapper::toLinkDto)
+                        .map(linkMapper::responseMap)
                         .collect(Collectors.toList()));
     }
 
     @Override
-    public CompletableFuture<List<LinkDto>> updateLinkEnableBfd(LinkEnableBfdDto link) {
+    public CompletableFuture<List<LinkDto>> bfdPropertiesWrite(
+            NetworkEndpoint source, NetworkEndpoint dest, boolean isEnabled) {
+        BfdProperties properties;
+        if (isEnabled) {
+            properties = bfdPropertiesDefault;
+        } else {
+            properties = BfdProperties.DISABLE;
```

Comment:
as for me, it is a little bit unusual that the `ed` ending is missing (`ed -> disabled`). we almost everywhere(except roundTrip) use the `ed` ending (i.e. `failed, activated, discovered`)

Reply:
renamed